hadoop-tools/hadoop-azure/dev-support/testrun-scripts/runtests.sh

```diff
@@ -126,39 +177,95 @@ do
    esac
 done
 
+## SECTION: CHECK FOR FNS+BLOB XML FILE
```

Comment:
Move this section and function to testsupport.sh along with other utility functions

Reply:
Right, makes sense